tinywallet/tinywallet/qutilities.py

```diff
@@ -65,7 +65,7 @@ def makeThread(self, func, callback=None, *args, **kwargs):
         thread.start()
         newThreadList = []
         for oldThread in self.threads:
-            if not oldThread.isFinished():
+            if not oldThread.completed:
```

Comment:
Relying on the `finished` signal here was problematic for recursively created threads with connection type `Qt.DirectConnection`. We are less interested in whether the `QThread.run` method has completed, and more interested in whether the callback function has been called.